common/utils/errors.js

```diff
@@ -199,6 +199,10 @@ export function defaultFormatGraphQLApiError(graphQLError, store) {
         return "Vous ne pouvez pas enregistrer des activités en dehors de votre période de rattachement.";
       case "ACTIVITY_OUTSIDE_EMPLOYMENT_ADMIN":
         return "Vous ne pouvez pas enregistrer des activités en dehors de la période de rattachement du salarié sélectionné.";
+      case "USER_NOT_EMPLOYED_BY_COMPANY_ANYMORE_EMPLOYEE":
+        return "Vous ne pouvez pas enregistrer des activités si vous n'êtes plus rattachés à la société.";
```

Comment:
:green_circle: On ne met pas au pluriel si c'est du vouvoiement, si ? :thinking: 
```suggestion
        return "Vous ne pouvez pas enregistrer des activités si vous n'êtes plus rattaché à la société.";
```